src/platform/mhv/api/mocks/index.js

```diff
@@ -32,14 +32,17 @@ const allergies = require('./medical-records/allergies');
 const acceleratedAllergies = require('./medical-records/allergies/full-example');
 const vaccines = require('./medical-records/vaccines');
 const vitals = require('./medical-records/vitals');
+const acceleratedVitals = require('./medical-records/vitals/accelerated');
 
 const responses = {
   ...commonResponses,
-  'GET /v0/user': user.defaultUser,
+  'GET /v0/user': user.acceleratedCernerUser,
```

Comment:
@mdewey can we keep `defaultUser` or add other mhv all other services to this `acceleratedCernerUser` so that mocks are still functional with the codespaces deploy?